File: projects/python_part_1/tip_calc.py
#tip calculator

bill = int(input("Total bill amount: "))
service = input("Level of service (good, bad, or fair): ")
service = service.lower()

if service == "good":
    tip = bill * .20
    total = bill + tip
    print("Tip amount: ", tip)
    print("Total amount: ", total)
elif service == "bad":
    tip = bill * .10
    total = bill + tip
    print("Tip amount: ", tip)
    print("Total amount: ", total)
elif service == "fair":
    tip = bill * .15
    total = bill + tip
    print("Tip amount: ", tip)
    print("Total amount: ", total)

File: javascript/tip_calc.js
var bill
var service
var tip
var total

function tipCalc (bill, service) {
  if (service == 'good') {
    tip = bill * 0.2;
    total = bill + tip;
    console.log("Bill amount: $" + bill);
    console.log("Recommended Tip: $" +  tip);
    console.log("Total: $" +  total);
  }
  else if (service == 'fair') {
    tip = bill * 0.15;
    total = bill + tip;
    console.log("Bill amount: $" + bill);
    console.log("Recommended Tip: $" +  tip);
    console.log("Total: $" +  total);
  }
  else if (service == 'bad') {
    tip = bill * 0.1;
    total = bill + tip;
    console.log("Bill amount: $" + bill);
    console.log("Recommended Tip: $" +  tip);
    console.log("Total: $" +  total);
  }
}


tipCalc(10, 'good');
